src/command-line/index.js

```diff
@@ -7,7 +7,12 @@ var fsextra = require("fs-extra");
 var path = require("path");
 var Helper = require("../helper");
 
-program.version(pkg.version, "-v, --version");
+program.version({
+	// Hack to defer execution of Helper.getGitCommit()
+	toString() {
+		return Helper.getGitCommit() ? Helper.getGitCommit() : pkg.version;
+	}
```

Comment:
No, I'm deferring execution of `toString` until `commander` [wants to use it](https://github.com/tj/commander.js/blob/v2.9.0/index.js#L824).

The overhead of spawning a child process to get the git SHA is pretty much non-existent though, but it feels a bit unnecessary to do it for every single command (especially since it'll fail since most lounge installations are done through `npm`, I assume).